spt.c

```diff
@@ -80,16 +80,28 @@ void
 remaining_time(int sigint)
 {
 	char buf[17];
-	if (signal(SIGUSR1, SIG_IGN) != SIG_IGN)
-		signal(SIGUSR1, remaining_time);
+	//if (signal(SIGUSR1, SIG_IGN) != SIG_IGN)
+	//	signal(SIGUSR1, remaining_time);
```

Comment:
Sorry for commenting on a closed PR.

As of manual for signal(2) say, it behavior varies across UNIX versions. I think the need to set the handler again in the handler itself is one of these variant behaviors.

It should be better to look at sigaction(2) for portability and consistency's sake. Would you be interested in such patch ?